### PR TITLE
Button/native: Properly Implement Disabled and Loading states of Button

### PR DESCRIPTION
**Pull request by kashishgrover**

### CODE REVIEW

**Comment by kamleshchandnani** [comment]
Deploy preview for *leaf-ui* processing.

Built with commit c80868ac76528ede21e95f87ff7a450634cdf89e

https://app.netlify.com/sites/leaf-ui/deploys/5bae2ca582d3f1126bd55c60

File: src/Button/native/Button.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ import Text from '../../Text/native';
 
 const styles = {
   buttonColor(props) {
+    if (props.kind === 'filled' && props.disabled) {
+      return `${props.theme.color[props.color]}aa`;
+    }
```
Comment (lakshyaranganath) [inline src/Button/native/Button.js]: you should be doing the opacity even if it's not a filled button

Comment (kashishgrover) [reply src/Button/native/Button.js]: wokay

File: src/Button/native/Button.js [hunk 2]
```diff
@@ -30,6 +48,15 @@ const styles = {
     }
     return props.theme.px(10);
   },
+  textColor(props) {
+    if (props.kind === 'filled') {
```
Comment (kamleshchandnani) [inline src/Button/native/Button.js]: `if (props.kind === 'filled' && !props.disabled)` ?

Comment (kashishgrover) [reply src/Button/native/Button.js]: disabled filled and enabled filled both will have white input as of now 🤔

**Pull request merged**